### DISCUSSION TITLE
Daily Payout | Monopoly Clause

### DISCUSSION THREAD

**Discussion by astrokin-app** [kinfoundation/kin-rfcs#114] [_ Archive - KRE 3.1 Discussion]
The KRE 3.1 can be advantageous for a greater number of applications, why not. Although, without trying this model and having statistics for a several months of payment it's difficult to say, if it will solve the problem of closed monopoly. 

The contradiction is about giving more to the apps that create more spends, create earns to make in-app wallets balances higher, and at the end reducing the daily payout from 250M to 220M. It seems to me to be the perfect example of "I give you with one hand and I take back with another".

Even with the new rules announced, I hardly believe that a new developer who will join the ecosystem, has a chance to earn as much as an app from top 3 or even a half of that amount. So the list will always look like: we have 3 apps, 3 giants of the ecosystem, and a bunch of amateurs. Because let’s be clear, what the majority of apps receive right now - it’s nice if your app is your hobby. But if it’s your work for living, you are in trouble. And the worst is that it doesn’t matter how hard you try.
Thanks!

**Comment by wmougayar** [comment]
Paying for upvotes is a sure way to discredit this comment, in addition to making claims that are not true.  
The KRE's evolution is specifically addressing a more fair distribution of rewards, especially for newcomers. And the justification for the cap adjustment has been explained in the proposal. 
We will provide a simulation calculator soon, where you could plug activity numbers and get a real KRE estimate.

**Reply by skyf0xx** [reply]
Kin's [whitepaper](https://kin.org/wp-content/uploads/Kin_Whitepaper_V1_English.pdf) detailed an inflation schedule, where the amount of Kin disbursed by the KRE each year reduces. Even though the amount of Kin is reduced every year, the USD value developers received has consistently gone upwards, partly because of this reduced supply.

This is not very different from other protocols such Bitcoin which halven periodically. Although miners get 'less bitcoin, they receive more USD. Naturally, the value of USD will depend on a bear or bull market, but it is still on a consistent trend upwards.

<img width="768" alt="image" src="https://user-images.githubusercontent.com/2501079/151732543-fdb2b8e6-7ace-49d7-9699-009866f5a7c4.png">

**Reply by skyf0xx** [reply]
In terms of smaller apps, we have also updated the Monopoly clause to help new apps start earning Kin. However, they need to make use of that starting amount to distribute to their users so they can move up the rewards ladder.

**Reply by skyf0xx** [reply]
@astrokre Perhaps something else to mention is, we proposed adding a curve on top of the [Monopoly Clause](https://github.com/kinecosystem/rewards-engine/blob/b63e38c3fd0eb110f58c85340bd561ee8593af12/current-KRE.md#monopoly-clause) which will improve the distribution to apps. However, the earning figures displayed are Pre Monopoly Clause (so apps understand how each stat affects earnings).

The Monopoly Clause states:
> No single developer will receive more than 66.67% of the KRE payout for a single day and any developer that would have received more than 50% of the payout will have their payout adjusted. No two developers will receive more than 90% of the KRE's payout for a single day. In both cases, residual payouts will be redistributed proportionally to other developers.
